aws-lc-rs/src/cipher.rs

```diff
@@ -243,41 +351,54 @@ pub enum OperatingMode {
 /// # let key = UnboundCipherKey::new(&AES_128, key_bytes)?;
 /// # let mut encrypting_key = EncryptingKey::ctr(key)?;
 /// #
-/// let context: CipherContext = encrypting_key.encrypt(&mut in_out_buffer)?;
+/// let context: DecryptionContext = encrypting_key.encrypt(&mut in_out_buffer)?;
 /// let iv_bytes: &[u8] = (&context).try_into()?;
 /// assert_eq!(16, iv_bytes.len());
 /// #
 /// #    Ok(())
 /// # }
 /// ```
-///
-///
-#[non_exhaustive]
-pub enum CipherContext {
-    /// A 128-bit Initialization Vector.
-    Iv128(FixedLength<IV_LEN_128_BIT>),
+pub struct DecryptionContext(CipherContext);
 
-    /// No input to the cipher mode.
-    None,
```

Comment:
I removed `None`, since we don't support any modes currently that would return this value.